core/src/blocktree.rs

```diff
@@ -1094,11 +1214,27 @@ impl Blocktree {
             }
         }
 
-        let (recovered_data, recovered_coding) = self
+        let (recovered_data, recovered_coding) = match self
             .session
-            .reconstruct_blobs(&mut blobs, present, size, start_idx, slot)?;
+            .reconstruct_blobs(&mut blobs, present, size, start_idx, slot)
+        {
+            Ok(x) => x,
+            Err(e) => {
+                error!(
```

Comment:
might be good to have an erasure metric here...

Reply:
I'll add one before merging